web-services-sample-app/ShowWeatherVC.swift

```diff
@@ -0,0 +1,114 @@
+        let myUrl = NSURL(string: urlWithParams.stringByAddingPercentEscapesUsingEncoding(NSUTF8StringEncoding)!)
+        let request = NSMutableURLRequest(URL:myUrl!);
+        request.HTTPMethod = CommonConstants.WebServices.GET_REQUEST
+        let task = NSURLSession.sharedSession().dataTaskWithRequest(request) {
+            data, response, error in
+            if error != nil
+            {
+                print("error=\(error)")
+                return
+            }
+            let responseString = NSString(data: data!, encoding: NSUTF8StringEncoding)
+            print("responseString = \(responseString)")
+            do {
+                    if let convertedJsonIntoDict = try NSJSONSerialization.JSONObjectWithData(data!, options: []) as? NSDictionary
+                    {
+                        print(convertedJsonIntoDict)
+                        self.parseDictionary(convertedJsonIntoDict)
+                    }
+                }
+                
+             catch let error as NSError {
+                print(error.localizedDescription)
+            }
+        }
+        task.resume()
+    }
+   
+    /** This function stores value of specific key in separate variables. The local variables are then converted to string to store them into model's object. The function also calls the displayWeatherDetails() function
+     * \param dict of NSDictionary type
+     * \returns Nothing
+     */
+    func parseDictionary(dict: NSDictionary) -> Void {
+        if let latitude = dict[CommonConstants.DictionaryKey.COORDINATES]?[CommonConstants.DictionaryKey.LATITUDE] as? Float {
+            weather.latitude = String(latitude)
+        }
+        if let longitude = dict[CommonConstants.DictionaryKey.COORDINATES]?[CommonConstants.DictionaryKey.LONGITUDE] as? Float {
+            weather.longitude = String(longitude)
+        }
+        if let temp_max = dict[CommonConstants.DictionaryKey.WEATHER_DETAILS_MAIN]?[CommonConstants.DictionaryKey.TEMP_MAX] as? Float {
+            weather.max_temperature = String(self.convertToCelsius(temp_max))
+        }
+        if let temp_min = dict[CommonConstants.DictionaryKey.WEATHER_DETAILS_MAIN]?[CommonConstants.DictionaryKey.TEMP_MIN] as? Float {
+            weather.min_temperature = String(self.convertToCelsius(temp_min))
+        }
+            self.displayWeatherDetails()
+    }
+    
+    /** This function converts the temperature from fahrenheit to degree celsius
+     * \param temperature of Float type. this parameter is received in Fahrenheit
+     * \returns convertedTemperature. convertedTemperature is in degree celsius
+     */
+    func convertToCelsius(temperature: Float) -> Float {
+        let convertedTemperature = ((temperature - 32)/1.8)
+        return convertedTemperature
+    }
+    
+    /** This function populates the textfields and displays information each text field is populated to display the required information
+     * \param None
```

Comment:
If no parameters , dont write the parameter list as Nothing , omit it all together.
If return is Void, mention as Void avoid using Nothing